build.gradle

```diff
@@ -542,27 +545,59 @@ task bundle(type: Zip) {
         rename 'GATKConfig.properties', 'GATKConfig.EXAMPLE.properties'
     }
 
-    from("${buildDir}/${pythonPackageArchiveName}")
-    // When including gatkcondaenv.yml file in the release bundle, strip off the
-    // 'build/' prefix used for the location of the Python package archive.
-    from("scripts/gatkcondaenv.yml", {
-        filter { line -> line.replace("build/${pythonPackageArchiveName}", pythonPackageArchiveName) }
-    })
+    from("$buildDir/$pythonPackageArchiveName")
+    from("$buildDir/$gatkCondaYML")
+    from("$buildDir/$gatkCondaIntelYML")
     into(baseName)
 
     doLast {
         logger.lifecycle("Created GATK distribution in ${destinationDir}/${archiveName}")
     }
 }
 
-task createPythonPackageArchive(type: Zip) {
+// Create the python package archive file, and two GATK conda environment yml files from the template
+// (one for standard GATK and one for running GATK with Intel hardware).
+task condaEnvironmentDefinition() {
+    dependsOn 'pythonPackageArchive'
+    inputs.file gatkCondaTemplate
+    outputs.dir buildDir
 
+    doLast() {
+        def standardCondaBindings = [
+                "condaEnvName":"gatk",
+                "condaEnvDescription" : "Conda environment for GATK Python Tools",
+                "tensorFlowDependency" : "tensorflow==1.4.0"]
+        createFileFromTemplate(gatkCondaTemplate, standardCondaBindings, new File("$buildDir/$gatkCondaYML"))
+
+        def intelCondaBindings = [
+                "condaEnvName":"gatkintel",
+                "condaEnvDescription" : "Conda environment for GATK Python Tools running with Intel hardware acceleration",
+                "tensorFlowDependency" :
+                        "https://anaconda.org/intel/tensorflow/1.4.0/download/tensorflow-1.4.0-cp36-cp36m-linux_x86_64.whl"]
+        createFileFromTemplate(gatkCondaTemplate, intelCondaBindings, new File("$buildDir/$gatkCondaIntelYML"))
+    }
+    doLast {
+        logger.lifecycle("Created conda environment yml files in: $buildDir")
+    }
+
+}
+
+// Populate a file based on a template, substituting placeholders with values from a bindings map.
+def createFileFromTemplate(sourceTemplateFile, variableBindings, targetFile) {
```

Comment:
There's some baked in stuff in the copy action that can do substitutions from a template (ex: http://mrhaki.blogspot.com/2010/10/gradle-goodness-parse-files-with.html).  

That might be more canonical for gradle, but I the way you're doing it is probably more understandable.  I just want to mention it incase you didn't know about it.

Reply:
I didn't know you could do that - I think we should prefer declarative tasks over functions where possible, so I switched to using two copy tasks with `expand`, and one target task with the other two as dependencies.